Sources/Common/Core/PriorityQueue/index.js

```diff
@@ -0,0 +1,61 @@
+import macro from 'vtk.js/Sources/macro';
+
+// ----------------------------------------------------------------------------
+// vtkPriorityQueue methods
+// ----------------------------------------------------------------------------
+
+function vtkPriorityQueue(publicAPI, model) {
+  // Set our classname
+  model.classHierarchy.push('vtkPriorityQueue');
+
+  publicAPI.push = (priority, element) => {
+    // naive algo
+    model.elements.push({ priority, element });
+    model.elements.sort(
+      ({ priorityLeft }, { priorityRight }) => priorityLeft - priorityRight
+    );
+  };
+
+  publicAPI.pop = () => {
+    if (model.elements.length > 0) {
+      const { element } = model.elements[0];
+      model.elements.shift();
```

Comment:
```
const { element } = model.elements.shift();
```
?

Reply:
We don't want to return the priority, so we only return the element.

Reply:
I know, but with my suggestion, you can combine 2 lines (21 &22) in just one line.

Alternatively:
```return model.elements.shift().element;```